azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/manager/PermissionManager.kt

```diff
@@ -58,16 +70,26 @@ internal class PermissionManager(
         }
     }
 
+    private fun createPhonePermissionRequest() {
+        if (getPhonePermissionState(activity) == PermissionStatus.NOT_ASKED) {
+            phonePermissionLauncher.launch(Manifest.permission.READ_PHONE_STATE)
```

Comment:
Doesn't this need to be added to the Android manifest ?

Reply:
It is in there as well, but it's a runtime permission on newer version of android, compile time old versions.